modular_skyrat/modules/modular_implants/code/soulcatcher/soulcatcher_component.dm

```diff
@@ -190,14 +192,12 @@ GLOBAL_LIST_EMPTY(soulcatchers)
 	to_chat(new_soul, span_doyourjobidiot("You have entered a soulcatcher, do not share any information you have received while a ghost. If you have died within the round, you do not know your identity until your body has been scanned, standard blackout policy also applies."))
 	to_chat(new_soul, span_notice("While inside of a soulcatcher, you are able to speak and emote by using the normal hotkeys and verbs, unless disabled by the owner."))
 	to_chat(new_soul, span_notice("You may use the leave soulcatcher verb to leave the soulcatcher and return to your body at any time."))
-
```

Comment:
keep the spacing here. The `to_chat` procs aren't really directly connected with the following code, so it doesn't make sense to have them bundled together.

Reply:
yeah that was a mistake